backend/tracim_backend/command/__init__.py

```diff
@@ -97,3 +99,14 @@ def get_parser(self, prog_name: str) -> argparse.ArgumentParser:
             default=False,
         )
         return parser
+
+    def changeDirectory(self) -> None:
+        directoryPath = os.path.dirname(__file__)
+        if len(directoryPath) > 30 and directoryPath[-30:] == "backend/tracim_backend/command":
```

Comment:
What is the name of the file used ? If I use prod.ini, is it still working ?

Reply:
and/or where we can set config file name ?

Reply:
the __file__ is the path to the file executed, backend/tracim_backend/command/__init__.py here, the config file name isn't link to that

Reply:
the config file name can be set with the -c option ( in that case a relative path from another place that the on we will move to execute tracimcli will not work)
or use whet we be developed here : https://github.com/tracim/tracim/issues/6312

Reply:
ok